### ISSUE TITLE
Can't get my medias as Tweets

### ISSUE THREAD

**Issue by jokoast** [linvi/tweetinvi#1011]
Hi, 

I did a little script to unfollow everyone, block/unblock everyone, delete my tweets, unfav and unretweet all BUT even if the "Tweets" tab and the "Tweets and Answers" tab are empty, the "Medias" one is full with medias i posted as answers. 

I can't manage to get them (to delete them).

**Comment by linvi** [comment]
This is interesting. Let me rephrase your use case here.

``` c#
// lets consider 2 users (a and b)
// user a publishes "Hello"
// user b replies to "Hello" with "How are you?" and an image
// user b deletes the reply ("How are you?" and an image)
// user b Medias tab still has the tweet and the image?
```

Could you please confirm?

**Comment by jokoast** [comment]
Nope, i'll rephrase too.
```
// lets considers 2 users (a and b [me])
// user a publishes "Hello"
// user b replies with a gif
```

I can't fetch the reply with gif only in the `Timeline.GetUserTimeline(userId, timelineRequestParameters)`

**Comment by linvi** [comment]
> I can't fetch the reply with gif only in the Timeline.GetUserTimeline(userId, timelineRequestParameters)

What is the `userId` (a or b)?
Do you experience something different when the reply contains text without the gif?

**Comment by jokoast** [comment]
As i deleted everything and it remains only the gif replies. I assume that the others are deleted.
To be more accurate, it remains the texte+gif and the gif only replies.

**Comment by linvi** [comment]
Hello there,

I think this is probably an issue with Twitter API.
I would suggest you report this on twittercommunity website here :

https://twittercommunity.com/

In terms of deleting tweets the endpoint is very straightforward and there is nothing I could do to change how the replied is destroyed from Tweetinvi.

Please link your ticket here so that I can follow up or comment and help.

Thanks,
Linvi
